qbraid_algorithms/qpe/phase_est.py

```diff
@@ -34,15 +30,18 @@
 
 
 class PhaseEstimationLibrary(GateLibrary):
-    '''
+    """
```

Comment:
We don't need this here - 

** Iterative phase estimation will be supported via
    the Rodeo package. This library supports both static and time-dependent Hamiltonians.**